src/util.js

```diff
@@ -0,0 +1,34 @@
+/**
+   Public util functions for the SDK.
+ */
+
+const isObject = val => Object.prototype.toString.call(val) === '[object Object]';
```

Comment:
Comment: We do have lodash dependency, so we could use also _.isPlainObject (https://lodash.com/docs/4.17.15#isPlainObject) but I don't know what's the advantage/disadvantage vs this implementation